edb/schema/functions.py

```diff
@@ -965,19 +965,19 @@ class Function(CallableObject, VolatilitySubject, s_abc.Function,
         qlast.Language, default=None, compcoef=0.4, coerce=True)
 
     from_function = so.SchemaField(
-        str, default=None, compcoef=0.4, introspectable=False)
+        str, default=None, compcoef=0.4)
 
     from_expr = so.SchemaField(
-        bool, default=False, compcoef=0.4, introspectable=False)
+        bool, default=False, compcoef=0.4)
 
     force_return_cast = so.SchemaField(
-        bool, default=False, compcoef=0.9, introspectable=False)
+        bool, default=False, compcoef=0.9)
 
     sql_func_has_out_params = so.SchemaField(
-        bool, default=False, compcoef=0.9, introspectable=False)
+        bool, default=False, compcoef=0.9)
 
     error_on_null_result = so.SchemaField(
-        str, default=None, compcoef=0.9, introspectable=False)
+        str, default=None, compcoef=0.9)
```

Comment:
Does this mean that all these fields are exposed to the `schema::`?

Reply:
Or we now only expose what's specified in `schema.edgeql` and the rest is automatically hidden?

Reply:
NVM, I should have started with reviewing individual commits.